documentRoot/userSites.php

```diff
@@ -37,7 +32,7 @@
 			else {
 				$filelist = "<tr><th>Id</th><th>Website Name</th><th>Created By</th><th>Created On</th><th>Is Domain?</th><th>Disable?</th></tr>";
 				while($row = mysqli_fetch_array($result)){
-					$form = '<form method="post" action="includes/siteDisable.php"><input type="hidden" name="siteID" value="'.$row['site_id'].'">';
+					$form = '<form method="post" action="includes/siteDisable.php"><input type="hidden" name="siteID" value="'.$row['website_name'].'">';
```

Comment:
I think it’s safer to use the site id here because it’s unique unlike the website name that can be reused over time.

Reply:
On that note, we should add a check to see if a website name is already under use when a iser creates a website.

Reply:
I changed it from site_id to website_name because using site_id breaks a lot of the handlers for the site manager. I can change everything to use site_id, but I think that's a bad idea, Since apache can't host two sites with the same name, website_name should be unique, and it makes more sense to use website_name for all the apache config handling.

Reply:
Making sure that the site name is unique should probably be done though.

Reply:
Uploading a site will now fail if the name is not unique

Reply:
Why does site_id break handlers? It worked fine for me when I was testing it.

Reply:
There's some inconsistency with how sites are named when uploaded vs enabled/disabled. When enabling a site using site_id, PHP tries to enable a conf file with that same siteID, which is an issue because the conf files use site_name. To use site_id, we'd also have to change how site upload works, and siteEnable, siteDisable, and siteDelete would become more complex.

In short, with how it's currently designed to operate, using website_name just makes everything a little bit simpler by not requiring a database connection for every step.

If you feel very strongly about this, and can give a good reason why site_id works better than website_name, I can make it work, but it would require re-writing how a lot of these PHP files work.

Reply:
I was just curious why you went with website_name. Since the conf files use the website_name (which makes sense), I think it's better to leave it that way